config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -127,6 +127,85 @@ periodics:
       - org: kubernetes
         slug: release-managers
 
+# Copy artifacts to mirrors (periodic drift correction)
+- interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  max_concurrency: 1
+  name: ci-k8sio-file-promo-mirrors
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: main
+  spec:
+    serviceAccountName: k8s-infra-promoter
+    containers:
+    - name: promote-to-mirrors
+      # TODO(justinsb): replace with released image once this is working
+      # Curently lacking S3 support - at least
+      #image: registry.k8s.io/artifact-promoter/kpromo:v3.4.11-1
+      image: gcr.io/k8s-staging-artifact-promoter/kpromo:v20230209-v3.4.11-54-ga7bc54b
```

Comment:
You might want to pin a digest, since more people have access to push over a staging tag but they cannot alter a digest.

`sha256:c075e38f1aba081824a3afc1cce63f1b0571f201de9b72397ab1000bb627420b` currently